.teamcity/builds/apiReferences/kotlinx/metadataJvm/KotlinxMetadataJvmBuildApiReference.kt

```diff
@@ -81,3 +53,11 @@ object KotlinxMetadataJvmBuildApiReference : BuildType({
         dependsOnDokkaTemplate(KotlinxMetadataJvmPrepareDokkaTemplates, "$LIB_DIR/dokka-templates")
     }
 })
+
+fun removeVPrefix(input: String): String {
+    return if (input.startsWith("v")) {
```

Comment:
🤔 is it necessary? just `return input.removePrefix("v")` doesn't work?

Reply:
thanks, fixed.